src/common/logger.ts

```diff
@@ -1,6 +1,6 @@
 import debug from "debug";
 
-if (!process.env.JEST_WORKER_ID) debug.enable("*"); // enable log outputs if not running in jest
+debug.enable("api-notarize-healthcerts:*,transient-storage-lib:*");
```

Comment:
Btw is this equivalent to setting `DEBUG=api-notarize-healthcerts:*` in package.json?

Reply:
I will revert the changes